package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "incrementals-publisher",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
?

Reply:
@timja released 1.3, but i guess manually so package-lock didn't get updated. when i did an npm install the lock file also updated.

Reply:
I didn’t realise it was in there as well =/.

Reply:
for node best way to bump version is `npm version $arg` but it honestly doesn't matter, i just felt it was worth doing